README.md

```diff
@@ -45,34 +45,23 @@ Read the current JupyterLab documentation on [ReadTheDocs](http://jupyterlab.rea
 
 ### Installation
 
-JupyterLab can be installed using [conda](https://docs.conda.io/en/latest/), [mamba](https://mamba.readthedocs.io/en/latest/) or [pip](https://docs.python.org/3.6/installing/index.html). For more detailed instructions, consult the [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
-
-Project installation instructions from the git sources are available in the [contributor documentation](CONTRIBUTING.md).
-
-### mamba and conda
-
-If you use `mamba` or `conda`, you can install it with:
-
-```shell
-mamba install -c conda-forge jupyterlab
-```
-
-or
+If you use [conda](https://docs.conda.io/en/latest/), [mamba](https://mamba.readthedocs.io/en/latest/), or [pip](https://docs.python.org/3/installing/index.html), you can install JupyterLab with:
 
 ```shell
+# conda
 conda install -c conda-forge jupyterlab
-```
-
-### pip
 
-If you use `pip`, you can install it with:
+# mamba
+mamba install -c conda-forge jupyterlab
 
-```shell
+# pip
 pip install jupyterlab
 ```
```

Comment:
I am afraid that novice users (which are a sizeable portion of the userbase, e.g. non-CS students learning statistics) will copy-paste all three commands. I find this surprisingly common, but I understand why this happens. As running both conda and pip in a single environment can cause issues, I would prefer to keep them separate with explicit "If you use X then do X" as it requires users to *think* about which option is appropriate for them.

I am fine with movin the "For more detailed instructions" paragraph down though.

Reply:
Good point. I have fixed this with a new commit. Let me know what you think!